app/menu-window.json

```diff
@@ -81,6 +81,12 @@
                 "accelerator": "CmdOrCtrl+Shift+S",
                 "command": "saveAs"
             },
+            {
+                "labelKey": "menu-save-autosave",
+                "command": "autoSaveOnFocusLost",
+                "type": "checkbox",
+                "checked": "autosave_on_focus_lost"
```

Comment:
This line must be fixed, otherwise the setting won't be saved to config.json as expected:

    "checked": "startup-commands:autoSaveOnFocusLost"